.github/workflows/wheel.yml

```diff
@@ -146,31 +141,15 @@ jobs:
           python -m pip install --upgrade pip
           python -m pip install cibuildwheel build
 
-      - name: "Restore cmdstan cache"
```

Comment:
I believe these lines are causing the rate limiting. They should not be removed and cmdstan should be downloaded via wget

Reply:
I suspect you're correct here. Downloading via wget makes `setup.py` much more complicated if we care about doing the right thing for users who didn't pre-download cmdstan, though. I'll look into it, though I think the test failures are not related to this

Reply:
^ These don't appear to be an issue anymore. All CI builds fail on the same test